packages/ubuntu_provision/test/services/page_config_service_test.dart

```diff
@@ -38,13 +37,16 @@ pages:
 
 MockConfigService createMockConfigService({
   String config = '',
-  ProvisioningMode mode = ProvisioningMode.standard,
+  ProvisioningMode? mode,
 }) {
   final mock = MockConfigService();
   final yaml = loadYaml(config)?['pages'] as YamlMap?;
-  when(mock.get('mode')).thenAnswer((_) async => mode.name);
+  when(mock.get('mode')).thenAnswer((_) async =>
+      mode != null ? mode.name : ProvisioningMode.standard.toString());
```

Comment:
One small detail:
```suggestion
      mode != null ? mode.name : ProvisioningMode.standard.name);
```
I think `toString()` would give `"ProvisioningMode.standard"` instead of `"standard"`